besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -401,12 +401,18 @@ private void detectJemalloc(final List<String> lines) {
         .ifPresentOrElse(
             t -> {
               try {
-                final String version = PlatformDetector.getJemalloc();
-                lines.add("jemalloc: " + version);
+                String jemallocEnabled = environment.get("BESU_USING_JEMALLOC");
+                if (jemallocEnabled.equals("true")) {
```

Comment:
```
if (Boolean.parseBoolean(jemallocEnabled)) {
 ..
}``` would cover case sensitive, null values etc.

Reply:
Also, the variable `t -> {` should be the value of `environment.get("BESU_USING_JEMALLOC")`. Instead of re-reading the value, its better to rename the variable `t` to `jemallocEnabled -> {`

Reply:
I moved the second ` environment.get("BESU_USING_JEMALLOC")`,